OPAL/ai/src/main/scala/org/opalj/ai/fpcf/analyses/LBFieldValuesAnalysis.scala

```diff
@@ -430,7 +439,12 @@ object FieldValuesAnalysis {
 
 object EagerLBFieldValuesAnalysis extends BasicFPCFEagerAnalysisScheduler {
 
-    override def requiredProjectInformation: ProjectInformationKeys = Seq(FieldAccessInformationKey)
+    override def requiredProjectInformation: ProjectInformationKeys = Seq(
+        FieldAccessInformationKey,
+        DeclaredMethodsKey,
+        DeclaredFieldsKey,
+        ContextProviderKey // TODO type mismatch (only in IDE) here?
```

Comment:
The reasons seems to be that both DeclaredMethodsKey and DeclaredFieldsKey have `Nothing` for the second generic type. With only one key having that, the type is still correctly inferred as `:< Object`, but with both, the bound is not correctly inferred anymore.
I don't fully understand this. `Nothing` is at the bottom of Scala's type hierarchy, so it is a subtype of AnyRef (Any even). The same problem occurs if both keys have `Null` (also at the bottom of the type hierarchy, but only subtype of AnyRef) as the generic type or one key has `Nothing` or `Null`, but not if only one key has `Nothing` or `Null` and the others have actual AnyRef types.